datastore/repo.go

```diff
@@ -24,15 +24,12 @@ import (
 
 type datastoreMgoRepo struct{}
 
-func (r datastoreMgoRepo) Add(dataItem DataItem) error {
+func (r datastoreMgoRepo) Add(item DataItem) error {
```

Comment:
Given my comments below, this could be:
```
func (r datastoreMgoRepo) Store(item DataItem) (updated bool, err error) {
    ...
    chgInfo, err := s.DB(mongo.DbName).C(mongo.DatastoreCollectionId).UpsertId(item.Key, item)
    updated = chgInfo.updated > 0
    ...
}
```

Reply:
Done